homeassistant/components/verisure.py

```diff
@@ -116,15 +121,24 @@ def setup(hass, config):
 
 def reconnect():
     """ Reconnect to verisure mypages. """
+    if not LOCK.acquire(blocking=False):
+        return
+    global PASSWORD_RETRIES
     try:
-        time.sleep(1)
+        time.sleep(10)
```

Comment:
Ok, I thougth each sensor update was threaded off. If it's the same I will not need a threadlock. 

I wanted to have a wait to not act as a brute force attack when the server denies access for some reason. Hard to know how much as a script we can behave.

I'll add a timer instead sine it makes more sence for a singel threaded solution.